man/systemd.network.xml

```diff
@@ -2296,6 +2296,16 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
 
     <variablelist class='network-directives'>
 
+      <varlistentry>
+        <term><varname>RelayTarget=</varname></term>
+        <listitem>
+          <para>Takes an IPv4 address, which must be in the format
+          described in
+          <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
```

Comment:
Anything that is resolvable should be OK. E.g. allowing the address to be specifies in `/etc/hosts` would be user friendly. But I guess just allowing literal addresses is enough.

Reply:
Yes, this is a nice feature